### DISCUSSION TITLE
Unbind with Alpine?

### DISCUSSION THREAD

**Discussion by ShawnDoe** [alpinejs/alpine#3576] [1. Help]
How can I unbind an event in Alpine? Or maybe what I am looking is an x-off? Basically, I want to remove the x-on after clicking it once. This is a x-on:click.outside and it doesn't seem to work to add the .once after the outside. In my .outside event can I unbind the event?

**Comment by ekwoka** [comment]
once and outside do work together.

some sample code would make sense of this question

**Comment by ShawnDoe** [comment]
I have a button with this working:
x-on:click.outside="jsToDotNet.closeProfileMenu"
This is in blazor so I am using the x-on. But when I add .once to the end of .outside, it doesn't work. Not even the once.
x-on:click.outside.once="jsToDotNet.closeProfileMenu"
That doesn't work.
